packages/amplication-git-utils/src/git/git-factory.ts

```diff
@@ -16,6 +17,10 @@ export class GitFactory {
         gitProvider = new GithubService(gitProviderArgs, logger);
         await gitProvider.init();
         return gitProvider;
+      case EnumGitProvider.Bitbucket:
```

Comment:
I hate switch/case, why not invoke two different functions

Reply:
the switch case is not a part of this PR. I just added a case. This PR is already big and I don't want to make big changes that are not related